disperser/apiserver/config.go

```diff
@@ -39,7 +39,7 @@ type PerUserRateInfo struct {
 
 type Allowlist = map[string]map[core.QuorumID]PerUserRateInfo
 
-type RateConfig struct {
+type Config struct {
```

Comment:
Wouldn't it be more clear to keep the name `RateConfig`? The fields here seem only relevant for rate limits, not for apiserver in general (which is the package this is under)

Reply:
it's generally better to have amore specific naming so it's more obvious what it's about